fdbclient/StorageServerInterface.h

```diff
@@ -394,12 +394,14 @@ struct GetStorageMetricsReply {
 	StorageMetrics available;
 	StorageMetrics capacity;
 	double bytesInputRate;
+	int64_t versionLag;
```

Comment:
Can you do the future-reader a favor by adding the definition of `versionLag` in comment?
Also in a fearless configuration, which tLog will be used to calculate the lag, the primary side or the remote side?

Reply:
The versionLag is difference between the latest version on the TLog the storage server is fetching and its current version